ProfitCurve chart missing Chart.js compenent

```
rrweb-plugin-console-record.js:142 Error: "scatter" is not a registered controller.
    at Fr._get (chart.js:5096:19)
    at Fr.getController (chart.js:5040:21)
    at sa.buildOrUpdateControllers (chart.js:5891:50)
    at sa.update (chart.js:5928:37)
    at sa.<anonymous> (chart.js:5687:48)
    at sa._doResize (helpers.dataset.js:602:16)
    at sa._resize (chart.js:5776:22)
    at sa.resize (chart.js:5747:18)
    at a (chart.js:6311:18)
    at sa.bindResponsiveEvents (chart.js:6323:13)
```

https://discord.com/channels/667551433503014924/1489964750610890752

--- a/src/ui/charts/PlanRepairCostChart.vue
+++ b/src/ui/charts/PlanRepairCostChart.vue
@@ -10,6 +10,8 @@
 		PointElement,
 		CategoryScale,
 		LinearScale,
+		LineController,
+		ScatterController,
 		type ChartData,
 		type ChartOptions,
 	} from "chart.js";
@@ -21,7 +23,9 @@
 		LineElement,
 		PointElement,
 		CategoryScale,
-		LinearScale
+		LinearScale,
+		LineController,
+		ScatterController
 	);
 
 	interface SeriesData {

